client/src/components/SubmissionTable.vue

```diff
@@ -15,9 +15,14 @@
         <div class="column">
           <h3 class="q-my-none">
             {{ $t(title) }}
+            <q-icon name="info">
+              <q-tooltip>{{ $t(tooltip, byline_opts) }}</q-tooltip>
```

Comment:
```suggestion
              <q-tooltip class="text-body1">{{
                $t(tooltip, byline_opts)
              }}</q-tooltip>
```
This will make the tooltip text 1rem (16px) and much easier to read.

Reply:
I updated the font size here, but it seems too big compared to the rest of the content on the page.

Reply:
In my opinion, it's the other way around: the application's default font size of 14px is too small. 

Rectifying this site-wide problem would be a massive scope increase for this task, so I would suggest just specifying an accessible font size for the tooltip for now, and then we can fix the small font size throughout the rest of the application later. 

What do you think?